src/components/date/date.js

```diff
@@ -447,27 +450,42 @@ class Date extends React.Component {
     return this.inputIconHTML('calendar');
   }
 
- /**
+  /**
+  * Returns the disabled array of days specified by props maxDate and minDate
+  *
+  * @method disabledDays
+  * @return {Array}
+  */
+  disabledDays() {
```

Comment:
Yeah the maxDate and minDate props were previously added as part of this PR: https://github.com/Sage/carbon/pull/342